documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -17,6 +17,18 @@ simplistic example program in the {example}[`session-example`] directory,
 which shows off all the "bits" you'll need to get your own program up and
 running.
 
+== Information about Hibernate ORM
+
+This document assumes some passing familiarity with Hibernate ORM or another
+implementation of JPA. If you've never used JPA before, that's OK, but you
+might need to refer to the following sources of information at some points
+in this text:
+
+- the http://hibernate.org/orm/documentation/5.5/[documentation for Hibernate ORM],
+- the https://jcp.org/aboutJava/communityprocess/mrel/jsr338/index.html[JPA 2.2 specification], or
+- https://www.manning.com/books/java-persistence-with-hibernate-second-edition[Java Persistence with Hibernate],
+  the third edition of the original book _Hibernate in Action_.
```

Comment:
You say third edition but it's actually the second edition

Reply:
It's the 3rd edition of "Hibernate in Action". It's the second named "Java Persistence with Hibernate". Confusing, I know.

Reply:
Can't we just not mentioned the exact edition and say `the updated version of the original book HIbernate in Action`? Or something similar

Reply:
If you insist.

Reply:
Yes, please